### PR TITLE
Brands

### PR DESCRIPTION
**Pull request by axrk**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     
         <!-- Define dependencies versions -->
-        <redscan-utils.version>4.2.1</redscan-utils.version>
+        <redscan-utils.version>4.2.2</redscan-utils.version>
```
Comment (Fisjkars) [inline pom.xml:17]: To update with the new version of Redscan-Utils (currently 4.3.0-SNAPSHOT)

File: src/main/java/com/michelin/cert/redscan/api/BrandController.java [hunk 2]
```diff
@@ -50,7 +54,7 @@ public class BrandAPIController {
      * Default constructor.s
```
Comment (Fisjkars) [inline src/main/java/com/michelin/cert/redscan/api/BrandController.java:54]: Typo

**Review by Fisjkars — CHANGES_REQUESTED** [review]

**Comment by Fisjkars** [comment]
Forgot to mention a point in my initial preview, you should add delete and update functions to complete the CRUD

**Comment by Fisjkars** [comment]
You could add the CI directly in this branch in order to validate checkstyle ;-)

**Comment by axrk** [comment]
Other CI tests should pass once Redscan-Utils branch is merged (for pagination)

**Pull request merged**